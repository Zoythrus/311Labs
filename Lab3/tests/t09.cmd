 text1.txt 6
