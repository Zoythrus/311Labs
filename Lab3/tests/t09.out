46 43 2 3 4 41 49 7 37 9 10 11 36 32 14 30 29 17 27 26 25 23 22 6 24 20 19 18 28 16 15 31 13 33 34 35 12 8 38 39 40 5 42 1 44 45 0 47 48 21 
